tool/src/main/java/org/wildfly/security/tool/CredentialStoreCommand.java

```diff
@@ -343,6 +350,23 @@ public void execute(String[] args) throws Exception {
             setStatus(ElytronTool.ElytronToolExitStatus_OK);
             return;
         }
+        if (cmdLine.hasOption(WEB_PARAM)) {
+            if (Desktop.isDesktopSupported()){
+                Desktop desktop = Desktop.getDesktop();
+                if (desktop.isSupported(Desktop.Action.BROWSE)){
+                    try {
+                        desktop.browse(new URI("https://docs.wildfly.org/26.1/WildFly_Elytron_Security.html#CredentialStore"));
```

Comment:
@Skyllarr Good point. Do you know what would be a good way to pass the parameter?

Reply:
@petrberan Sorry I might have used a wrong word. I did not mean adding a parameter to the elytron tool that would specify the version. That would get more complicated to specify the version when using the tool with EAP. Its okay to make a constant out of "https://docs.wildfly.org" and eg. DOCS_VERSION="27"

Reply:
@petrberan  Otherwise this PR LGTM, thank you!

Reply:
@Skyllarr I was also wondering how the version would pass from WFLY to WFCORE and then to ELY. Created a constants for the doc URI in each command class, should be alright now

Reply:
@fjuma When using tool with EAP, it is okay that it directs users to wildfly docs instead of EAP right?

Reply:
Let's see what @OndrejKotek thinks when he's back from PTO.

We also need to check with @OndrejKotek to see if he thinks this should be handled as an RFE since it introduces a new option for Elytron Tool.

Reply:
Regarding upstream docs in product. I'm not sure, asking higher instances. However, I think there should be a possibility to change the URLs easily, e.g. by building with a provided properties file with URLs.

Regarding having the WF version (and maybe even doc URL) hard-coded. Can this parametrization be moved a bit closer to releasing so that we don't forget to update to the latest? E.g. as property in pom.xml? Or as a properties file maintained along to WF documentation, referenced in pom.xml?

Regarding RFE process. The change is quite simple, however if we would go the way with different URLs for the product we needed some agreements and tests. Maybe still doable in scope of an JBEAP. Let's discuss this on the next Ely team meeting.